### ISSUE TITLE
Outdated `static.rust-lang.org` documentation from 2014

### ISSUE THREAD

**Issue by cyrgani** [rust-lang/rust#129328]
This was inspired by #129326 which referred to outdated documentation from 2014 on this domain: [link](https://static.rust-lang.org/doc/master/std/index.html).

I have not looked further yet, but it _seems_ as if this whole documentation part is no longer used and could be removed.

If it turns out to still be used for something, at least the documentation should marked as outdated to avoid further confusion.

**Comment by iamthad** [comment]
Yes, I'll admit that I didn't realize that I should be at `doc.rust-lang.org` instead of `static.rust-lang.org/doc`, and I missed the 1.0.0 version number at the top of the page.

**Comment by hkBst** [comment]
The link from you first message now redirects to the current docs, so I think this can be closed, @cyrgani.
